Pd 408 review accounts users and groups content (#2029)
truenas/documentation: content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md

By default, TruNAS hides the system built-in groups.
To see built-in groups, click the **Show Built-In Groups** toggle. The toggle turns blue and all built-in groups display. Click the **Show Built-In Groups** toggle again to show only non-built-in groups on the system.

## Adding a New Group

To create a group, go to **Credentials > Local Groups** and click **Add**.

{{< trueimage src="/images/SCALE/23.10/AddGroupGIDConfigSCALE.png" alt="Add Group Screen" id="2: Add Group Screen" >}}

Enter a unique number for the group ID in **GID** that TrueNAS uses to identify a Unix group.
Enter a number above 3000 for a group with user accounts or enter the default port number as the GID for a system service.

Enter a name for the group.
The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=).
You can only use the dollar sign ($) as the last character in a group name.

> **Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allow members of the group to act as root using the [sudo](https://www.sudo.ws/) command.

Reviewer: We don't want to say root here, these commands are there for non-root admin users so to comply with the government security requirement (FIPS). so you can say limited root-like permissions, but don't say the root user.

To allow sudo commands, use **Allowed sudo commands** or **Allowed sudo commands with no password** to list the executable path(s) to specific sudo commands allowed for group members.
<file>/usr/bin/</file> is the default location for command executables.

> Alternately, click the checkbox for **Allow all sudo commands** or **Allow all sudo commands with no password**.

Reviewer: style guide says to not say checkbox unless there is no other way, also when it is a checkbox use select, for example
```suggestion
Alternately, Select **Allow all sudo commands** or **Allow all sudo commands with no password**.
```

Exercise caution when allowing sudo commands, especially without password prompts.
It is recommended to limit this privilege to trusted users and specific commands to minimize security risks.

To allow Samba permissions and authentication to use this group, select **Samba Authentication**.

To allow more than one group to have the same group ID (not recommended), select **Allow Duplicate GIDs**.
Use only if absolutely necessary, as duplicate GIDs can lead to unexpected behavior.

## Managing Groups

> Click the <span class="material-icons">expand_more</span> arrow or anywhere on a row to expand that group and show the group management buttons.

Reviewer: Since the behavior of clicking on a row allows the user to not have to click the arrow, maybe we should just tell them to click anywhere on a row and do away with the reference to the expand arrow. This is a change from Anglefish and CORE where you had to click on the expand arrows.

{{< trueimage src="/images/SCALE/23.10/GroupsListedExpandedSCALE.png" alt="Expanded Group Screen" id="3: Expanded Group Screen" >}}

Use these buttons to add or remove group <i class="material-icons" aria-hidden="true" title="Members">group</i> [**Members**](#update-members), <i class="material-icons" aria-hidden="true" title="Edit">edit</i> [**Edit**](#edit-group) group configuration, or <i class="material-icons" aria-hidden="true" title="Delete">delete</i> **Delete** the group.

### Update Members

To manage group membership, go to **Credentials > Local Groups**, expand the group entry, and click **Members** to open the **Update Members** screen.

{{< trueimage src="/images/SCALE/23.10/GroupsManageMembersSCALE.png" alt="Update Members Screen" id="4: Update Members Screen" >}}

> To add user accounts to the group, select users and then click the right arrow <i class="fa fa-arrow-right" aria-hidden="true" title="Right Arrow"></i>.

Reviewer: Because you mention selecting multiple users after these two steps, make them single user instructions.
```suggestion
To add a user account to the group, select a user and then click the right arrow <i class="fa fa-arrow-right" aria-hidden="true" title="Right Arrow"></i>.
```

> To remove user accounts from the group, select users and then click the left arrow <i class="fa fa-arrow-left" aria-hidden="true" title="Left Arrow"></i>.

Reviewer: same comment above
```suggestion
To remove a user account from the group, select the user and then click the left arrow <i class="fa fa-arrow-left" aria-hidden="true" title="Left Arrow"></i>.
```

> Select multiple users by holding <kbd>Ctrl</kbd> while clicking each entry.

Reviewer: ```suggestion
To select multiple users, press<kbd>Ctrl</kbd> and click on each entry. 
```
